server/src/application/application.service.ts

```diff
@@ -11,6 +13,20 @@ export class ApplicationService {
     private applicationRepository: Repository<Application>
   ) {}
 
+  async getApplications(query: GetApplicationsDto): Promise<PaginationRO<Application>> | null {
+    const queryBuilder = this.applicationRepository.createQueryBuilder('app');
+
+    if (query.facilityName)
+      queryBuilder.andWhere('app.facilityName like :facilityName', {
+        facilityName: `%${query.facilityName}%`,
+      });
+    if (query.orderBy) queryBuilder.orderBy({ [`app.${query.orderBy}`]: query.order });
```

Comment:
brackets.

Reply:
Sorry Karthikk, what do you mean by brackets? You mean, after the IF statement or? The curly brackets? {} I thought you meant to use another overload of .orderBy that uses curly brackets, but I guess I misunderstood, thanks in advance!